src/level/scoreboard.h

```diff
@@ -32,6 +32,22 @@
 
 class CObject;
 
+/**
+ * \struct Score
+ * \brief Struct containing score of individual team and additional variables to allow sorting teams through different criteria
+*/
+struct Score
+{
+    int points = 0; //! Team score
+    float time = 0; //! Time when points were scored 
+};
+
+enum SortType
```

Comment:
New code should probably prefer using strongly-typed enums (`enum class`) over traditional C enums, but I think it can stay like that for now